view/lang/de/messages.po

```diff
@@ -7597,7 +7598,7 @@ msgstr "Neue Basis-URL"
 msgid ""
 "Change base url for this server. Sends relocate message to all DFRN contacts"
 " of all users."
-msgstr "Ändert die Basis-URL dieses Servers und sendet eine Umzugsmitteilung an alle DFRN Kontakte deiner Nutzer_innen."
+msgstr "Ändert die Basis-URL dieses Servers und sendet eine Umzugsmitteilung an alle DFRN Kontakte deiner NutzerInnen."
```

Comment:
@annando It's nothing to block this PR, but in general to change in the admin panel. The server relocation now sends the relocation to the Diaspora* contacts as well right?

Reply:
Yeah.